src/messages/manifestjson.js

```diff
@@ -296,6 +296,17 @@ export function corruptIconFile({ path }) {
   };
 }
 
+export const MANIFEST_THEME_LWT_ALIAS = {
+  code: 'MANIFEST_THEME_LWT_ALIAS',
+  message: i18n._('This theme LWT alias is deprecated.'),
+  // TODO: update the short link included in the description with one that
+  // points directly to a section which describes the changes needed.
+  description: i18n._(
+    'See https://mzl.la/1ZOhoEN (MDN Docs) for more information.'
```

Comment:
And make that link key something named like `mzl.la/mdn-lwt-alias-deprecation` ideally but that's optional, really.

Reply:
@EnTeQuAk I've replaced the link with one that points to https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/theme#Aliases, unfortunately it seems that if I try to customize the link then it becomes a bit.ly link instead of a mzl.la link (e.g. https://mzl.la/2T11Lkc is a mzl.la link that points to the above url, if I create a customized one it becomes http://bit.ly/mdn-lwt-alias-deprecation but http://mzl.la/mdn-lwt-alias-deprecation doesn't exist :-( ).

As additional side note, the additional docs about the deprecation are not in that section yet, the MDN issue for those docs is mdn/sprints#821.

Reply:
Ah, that's quite unfortunate. really hard, I don't know which is better, generally a customized link is more user friendly as a user kinda knows where he's getting to and to be very fair, mzl.la does safeguard the user somewhat but I don't really know if anyone notices :man_shrugging:

Reply:
> Ah, that's quite unfortunate. really hard, I don't know which is better, generally a customized link is more user friendly as a user kinda knows where he's getting to and to be very fair, mzl.la does safeguard the user somewhat but I don't really know if anyone notices man_shrugging

Yeah, it is unfortunate :-(
I ended to prefer the mzl.la one for consistency with the urls included in the other messages defined in `src/messages/manifestjson.js`.